.repo/sample-list-generator/src/fetch.ts

```diff
@@ -0,0 +1,70 @@
+import fetch from 'node-fetch';
+import path from 'path';
+import fs from 'fs/promises';
+import { JSDOM } from 'jsdom';
+import { TExtensionApiMap } from './types';
+
+const getApiNames = async () => {
+  const re =
+    /<tr><td><a class="link weight-medium" href=(\S+)>(\S+)<\/a><\/td><td class="reference-collapse type">/g;
+  const rawHtml = await (
+    await fetch('https://developer.chrome.com/docs/extensions/reference/')
```

Comment:
It'd be good to check for the error code here, otherwise we might parse a 503 error page...